flytekit/core/interface.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         inputs: typing.Optional[typing.Dict[str, Union[Type, Tuple[Type, Any]]]] = None,
         outputs: typing.Optional[typing.Dict[str, Type]] = None,
         output_tuple_name: Optional[str] = None,
+        docstring: Optional[Docstring] = None,
```

Comment:
mind updating the docstring (hah) for this new param below?

Reply:
Fixed